2.10/2.10.5/index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Title</title>
+</head>
+<body>
+### 2.10.5
+На странице появляется модалка с вопросом: "Сколько тебе лет?"
+Пользователь может ввести любые данные: число или текст. А дальше будет 2 пути развития:
+1) Если пользователь вводит число больше нуля, то потом выскакивает алерт с сообщением: "До пенсии осталось Х лет".
+Где Х - это вычисленное значение (пенсия в 60 лет).
+2) Если пользователь ввёл что-то другое, то должен выскочить алерт с сообщением: "Введенные данные неправильные,
+попробуйте еще раз".
+}
+<script>
+	let age = prompt('Сколько тебе лет?');
+
+	if (age > 0) {
+		alert(`До пенсии осталось ${60 - age} лет`);
```

Comment:
Please avoid any calculations inside a value, you can create a new variable with needed calculation